docker/lookaside.go

```diff
@@ -59,13 +66,16 @@ func configuredSignatureStorageBase(sys *types.SystemContext, ref dockerReferenc
 	}
 
 	topLevel := config.signatureTopLevel(ref, write)
+	var url *url.URL
 	if topLevel == "" {
```

Comment:
(Absolutely non-blocking: It would be a tiny bit more readable to start with `topLevel != ""`, so that the uses of `topLevel` are close together.)